<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindAccoundCBox</class>
 <widget class="QWidget" name="FindAccoundCBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>530</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Поиск</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Фильтр по ФИО и адресам</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Фамилия</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Адрес</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="search_adress">
           <property name="placeholderText">
            <string>Адрес дома</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEdit">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>№ Квартиры</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="search_name_2">
           <property name="placeholderText">
            <string>Фамилия</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QPushButton" name="search_two_column">
           <property name="text">
            <string>Поиск</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Навигация по поиску</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="0" colspan="4">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="1">
            <widget class="QLineEdit" name="search_name_3">
             <property name="placeholderText">
              <string>Введите строку поиска</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="typeSearchCBox">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="FindFirstButton">
           <property name="text">
            <string>Первая</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="FindPreviousButton">
           <property name="text">
            <string>Предыдущая</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="FindNextButton">
           <property name="text">
            <string>Следующая</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="FindLastButton">
           <property name="text">
            <string>Последняя</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="reset_filter">
        <property name="text">
         <string>Сбросить фильтр</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="search_id">
        <property name="toolTip">
         <string/>
        </property>
        <property name="placeholderText">
         <string>фильтр по идентификатору аккаунта</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="title">
         <string>Сортировка</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QComboBox" name="FirstField"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="FirstSortType"/>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="SecondField"/>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="SecondSortType"/>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>92</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Сортировка</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
